README.md

```diff
@@ -41,9 +41,12 @@ brew install move2kube
 
 To bring up UI version:
 
-* Change directory to an empty directory using say, `mkdir -p workspace && cd workspace`
-* Run `docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest`
-* Access the UI in `http://localhost:8080/`.
+```
+docker run --rm -it -p 8080:8080 -v "${PWD}/data:/move2kube-api/data" -v /var/run/docker.sock:/var/run/docker.sock quay.io/konveyor/move2kube-ui:latest
```

Comment:
Let's remove the docker.sock. its only for advanced usecases.

Reply:
done